druid/src/win_handler.rs

```diff
@@ -171,6 +173,14 @@ impl<'a, T: Data> SingleWindowState<'a, T> {
         self.do_layout(piet);
         piet.clear(self.env.get(theme::WINDOW_BACKGROUND_COLOR));
         self.do_paint(piet);
+
+        // schedule an idle call with the runloop if there are commands to process after painting is finished,
+        // that would trigger a new event/update pass.
+        if !self.command_queue.is_empty() {
+            let mut handle = self.state.handle.get_idle_handle().unwrap();
```

Comment:
I'm not sure under what conditions this can be `None`, but I would probably be defensive? Although we'd also prefer not to fail silently, so I might also log the failure :(

Reply:
Fixed 👍